.github/workflows/dotnetcore.yml

```diff
@@ -1,21 +1,21 @@
-name: eShopOnWeb Build and Test
-
-on: [push, pull_request, workflow_dispatch]
-
-jobs:
-  build:
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Setup .NET
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: '5.0.x'
-
-    - name: Build with dotnet
-      run: dotnet build ./eShopOnWeb.sln --configuration Release
-    
-    - name: Test with dotnet
-      run: dotnet test ./eShopOnWeb.sln --configuration Release
+name: eShopOnWeb Build and Test
```

Comment:
The change I made here was already done in https://github.com/dotnet-architecture/eShopOnWeb/pull/479. I was expecting this file to disappear after I fixed the conflicts. Not sure why it's still present in this PR without any changes.

Reply:
Want to just do a clean PR with the project file fixes?

Reply:
@ardalis Sure. Will open a new one.